src/shared/tpm2-util.c

```diff
@@ -133,6 +133,50 @@ static inline void Esys_Freep(void *p) {
                 sym_Esys_Free(*(void**) p);
 }
 
+static int tpm2_get_capability(
+                Tpm2Context *c,
+                TPM2_CAP capability,
+                uint32_t start,
+                uint32_t count,
+                TPMI_YES_NO *ret_more,
+                TPMU_CAPABILITIES *ret_capability_data) {
+
+        _cleanup_(Esys_Freep) TPMS_CAPABILITY_DATA *capabilities = NULL;
+        TPMI_YES_NO more;
+        TSS2_RC rc;
+
+        assert(c);
+
+        log_debug("Getting TPM2 capability 0x%04x property 0x%04x count %u.", capability, start, count);
```

Comment:
fully correct would be to format the uint32_t in the format tring as `… property 0x%04" PRIx32 " count…`

Reply:
ack